regression/cbmc-sequentialization/posix_semaphores/blocking_wait.desc

```diff
@@ -1,4 +1,4 @@
-FUTURE
+FUTURE requires_posix_only_headers
```

Comment:
I'd be careful with naming like this, AFAIK Windows is (or at least has been at some point) POSIX compliant.

Reply:
A standard install of Visual Studio does not make the POSIX header files we need available as part of its include path. Without these the build environment is does not have the POSIX compliance level we need. The windows API calls for threading are available through separate header and will have different naming conventions entirely. If a Windows based build environment were to be setup with the appropriate POSIX header files made available, then it should be possible to make these tests work on Windows.

Technically I believe these these would need a minimum of POSIX.1b for semaphores and POSIX.1c for thread creation and joining. The original POSIX.1 version would not be sufficient. But version POSIX.1-2001 onward would have everything we need.